addon/components/flexberry-maptoolbar.js

```diff
@@ -235,20 +234,38 @@ let FlexberryMaptoolbarComponent = Ember.Component.extend({
 
   /**
     Observes changes in {{#crossLink "FlexberryMaptoolbarComponent/leafletMap:propery"}}'leafletMap' property{{/crossLink}}.
-    Activates default map-tool when leafletMap initialized.
+    Activates default map-tool when leafletMap initialized and subscribes on flexberry-map:identificationFinished event.
 
     @method _leafletMapDidChange
     @private
   */
-  _leafletMapDidChange: Ember.on('didInsertElement', Ember.observer('leafletMap', function() {
-    if (Ember.isNone(this.get('leafletMap'))) {
+  _leafletMapDidChange: Ember.on('didInsertElement', Ember.observer('leafletMap', function () {
+
+    let leafletMap = this.get('leafletMap');
+    if (Ember.isNone(leafletMap)) {
       return;
     }
 
+    // Attach custom event-handler.
+    leafletMap.on('flexberry-map:identificationFinished', this.identificationFinished, this);
```

Comment:
Toolbar is bad choice for this logic. Tool bar doesn't know anithing about available tools and commands, about their specific events, i think flexberry-identify-panel is better plase to catch this event i translate it into conponent's action.